dataactvalidator/config/sqlrules/sqlRules.csv

```diff
@@ -88,7 +88,8 @@ C8,Unique FAIN/URI from file C exists in file D2,Unique FAIN/URI from file C exi
 C9,"Unique FAIN/URI from file D2 exists in file C, except D2 records with zero FederalActionObligation","Unique FAIN/URI from file D2 exists in file C, except D2 records with zero FederalActionObligation",y,award,warning,award_financial,c9_award_financial
 C11,"Unique PIID, ParentAwardId from file C exists in file D1 during the same reporting period.","Unique PIID, ParentAwardId from file C exists in file D1 during the same reporting period.",y,award_financial,warning,award_procurement,c11_cross_file
 C12,"Unique PIID, ParentAwardId from file D1 exists in file C during the same reporting period, except D1 records with zero FederalActionObligation","Unique PIID, ParentAwardId from file D1 exists in file C during the same reporting period, except D1 records with zero FederalActionObligation",y,award_procurement,warning,award_financial,c12_cross_file
-C14,If FAIN is not provided then provide URI,If FAIN is not provided then provide URI,n,award_financial,warning,,c14_award_financial
+C14,If FAIN is not provided then provide URI,"Must provide either FAIN, URI, or PIID",n,award_financial,fatal,,c14_award_financial_1
```

Comment:
I realize this text pre-dates your PR, but the `rule_description` here (2nd column) isn't right: `If FAIN is not provided then provide URI`

Since we now have D1 records in the mix, would it make more sense to re-word as `FAIR, URI, or PIID must be provided` (or just copy the current wording from the validation rules .csv, which is maybe what we did to begin with, and now it needs updated).

I know someone will likely take a final pass at the various error messages, but I want to make sure we're not confusing agencies in the meantime.